program-states/src/state/deposit_entry.rs

```diff
@@ -23,9 +23,10 @@ pub struct DepositEntry {
     pub voting_mint_config_idx: u8,
     // True if the deposit entry is being used.
     pub is_used: bool,
+    pub _reserved0: [u8; 32],
     pub _reserved1: [u8; 6],
 }
-const_assert!(std::mem::size_of::<DepositEntry>() == 32 + 32 + 8 + 8 + 1 + 1 + 6);
+const_assert!(std::mem::size_of::<DepositEntry>() == 48 + 32 + 32 + 8 + 8 + 1 + 1 + 6);
```

Comment:
It's always nice to have the same order in the size check as in the fields
```suggestion
const_assert!(std::mem::size_of::<DepositEntry>() == 48 + 32 + 8 + 8 + 1 + 1 + 32 + 6);
```